### ISSUE TITLE
Exception thrown when selecting "Icons" section

### ISSUE THREAD

**Issue by remonator** [TYPO3/styleguide#174]
# Bug Report

## Prerequisites

* [x] Can you reproduce the problem on TYPO3 v10.4 LTS?
* [ ] Can you reproduce the problem on TYPO3 v9.5 LTS?

## Description

Exception thrown when selecting "Icons" section:
```
Core: Exception handler (WEB): Uncaught TYPO3 Exception: #1437425804: Icon with identifier "warning-lock" is not registered"
```

## Steps to Reproduce

1. From the styleguide backend index page, click on "Icons" in the table on the left-hand side.

**Expected behavior:** [What you expected to happen]

"Icon" section opens.

**Actual behavior:** [What actually happened]

Exception is thrown

## Versions

Styleguide 10.0.0
TYPO3 10.4.10

You can get this information from executing `composer show` at the command line if your TYPO3 installation is composer based, otherwise the version number will be shown in the extension manager.

**Comment by lolli42** [comment]
Interesting. I'm unable to reproduce this issue with styleguide 10.0.7 and core 10.4.12.
Would you like to re-check this with the youngest v10 compatible styleguide, maybe this has been fixed meanwhile?
The 'warning-lock' icon is a standard core icon and i think it's always registered, it should be there, but core also had some changes in this area, so it would be good if this is reproducible for you with recent versions.

**Comment by remonator** [comment]
I can confirm the issue is resolved with TYPO3 10.4.12 and `styleguide` 10.0.7 👍

**Comment by lolli42** [comment]
Great! Thanks for feedback. Closing.
